.github/workflows/macos-ci.yml

```diff
@@ -49,8 +49,8 @@ jobs:
         run: brew install boost-python3 gtk+3 gtkglext sdl
 
       # The following Apple-provided libraries are deprecated:
-      # * OpenGL as of MacOS 10.14
-      # * GLUT as of MacOS 10.9
+      # * OpenGL as of macOS 10.14
+      # * GLUT as of macOS 10.9
```

Comment:
NOTE: All the MacOS 11 builds failed; MacOS 12 passed for all except including OpenAL with GCC if I understand the matrix right.

Reply:
Filed https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/812 for the macOS build failures. They all seem to be related to which version of Python Boost is trying to use.